lib/ansible/modules/remote_management/cpm/cpm_serial_port_config.py

```diff
@@ -0,0 +1,345 @@
+        if (existing_serial["serialports"][0]["portname"] != to_native(cpmmodule.params["portname"])):
+            total_change = (total_change | 1)
+            json_load = '%s,"portname": "%s"' % (json_load, to_native(cpmmodule.params["portname"]))
+    if cpmmodule.params["baud"] is not None:
+        if (existing_serial["serialports"][0]["baud"] != to_native(cpmmodule.params["baud"])):
+            total_change = (total_change | 2)
+            json_load = '%s,"baud": %s' % (json_load, to_native(cpmmodule.params["baud"]))
+    if cpmmodule.params["handshake"] is not None:
+        if (existing_serial["serialports"][0]["handshake"] != to_native(cpmmodule.params["handshake"])):
+            total_change = (total_change | 4)
+            json_load = '%s,"handshake": %s' % (json_load, to_native(cpmmodule.params["handshake"]))
+    if cpmmodule.params["stopbits"] is not None:
+        if (existing_serial["serialports"][0]["stopbits"] != to_native(cpmmodule.params["stopbits"])):
+            total_change = (total_change | 8)
+            json_load = '%s,"stopbits": %s' % (json_load, to_native(cpmmodule.params["stopbits"]))
+    if cpmmodule.params["parity"] is not None:
+        if (existing_serial["serialports"][0]["parity"] != to_native(cpmmodule.params["parity"])):
+            total_change = (total_change | 16)
+            json_load = '%s,"parity": %s' % (json_load, to_native(cpmmodule.params["parity"]))
+    if cpmmodule.params["mode"] is not None:
+        if (existing_serial["serialports"][0]["mode"] != to_native(cpmmodule.params["mode"])):
+            total_change = (total_change | 32)
+            json_load = '%s,"mode": %s' % (json_load, to_native(cpmmodule.params["mode"]))
+    if cpmmodule.params["cmd"] is not None:
+        if (existing_serial["serialports"][0]["cmd"] != to_native(cpmmodule.params["cmd"])):
+            total_change = (total_change | 64)
+            json_load = '%s,"cmd": %s' % (json_load, to_native(cpmmodule.params["cmd"]))
+    if cpmmodule.params["seq"] is not None:
+        if (existing_serial["serialports"][0]["seq"] != to_native(cpmmodule.params["seq"])):
+            total_change = (total_change | 128)
+            json_load = '%s,"seq": %s' % (json_load, to_native(cpmmodule.params["seq"]))
+    if cpmmodule.params["tout"] is not None:
+        if (existing_serial["serialports"][0]["tout"] != to_native(cpmmodule.params["tout"])):
+            total_change = (total_change | 256)
+            json_load = '%s,"tout": %s' % (json_load, to_native(cpmmodule.params["tout"]))
+    if cpmmodule.params["echo"] is not None:
+        if (existing_serial["serialports"][0]["echo"] != to_native(cpmmodule.params["echo"])):
+            total_change = (total_change | 512)
+            json_load = '%s,"echo": %s' % (json_load, to_native(cpmmodule.params["echo"]))
+    if cpmmodule.params["break_allow"] is not None:
+        if (existing_serial["serialports"][0]["break"] != to_native(cpmmodule.params["break_allow"])):
```

Comment:
If you change this param to be type `bool`, you'll need no convert these with `int()` when comparing. Since `existing_serial["serialports"][0]["break"]` is a text string and `cpmmodule.params["break_allow"]` would be a `bool`.